### PR TITLE
replace prints with appropriate log level.

### LINKED ISSUE #114: Replace `println` by `info` in logger
The codebase has too many `println`. We should use the sbt logger and pass it in from function to function.

### PR DESCRIPTION
**Pull request by erip**
Closes #114.

### CODE REVIEW

File: src/main/scala/Resolver.scala [hunk 1]
```diff
@@ -23,7 +23,7 @@ case class BintrayMavenRepository(
       Duration.Inf) match {
         case (201, _) =>
         case (_, fail) =>
-          println(fail)
```
Comment (jvican) [inline src/main/scala/Resolver.scala]: Why is this necessary?

Comment (erip) [inline src/main/scala/Resolver.scala]: Doesn't seem necessary. I'd be happy to remove it.

Comment (jvican) [inline src/main/scala/Resolver.scala]: Yes, please, do so :) In fact, you should do `log.error(fail)`. With the current statement, you will prevent the `throw new RuntimeException` from being executed, and that's the one that tells the user what really happened.

Comment (erip) [inline src/main/scala/Resolver.scala]: So to be clear, we want to `log.error(fail)` instead of removing it?

**Comment by erip** [comment]
This is going to require a bit of research on my part... thought this would be a quick(er) one. 😄 I'll finish it tonight.

File: src/main/scala/Bintray.scala [hunk 2]
```diff
@@ -60,11 +60,11 @@ object Bintray {
     BintrayCredentials.read(credsFile).fold(sys.error(_), _ match {
       case None =>
         if (prompt) {
-          println("bintray-sbt requires your bintray credentials.")
+          log.info("bintray-sbt requires your bintray credentials.")
```
Comment (travissarles) [inline src/main/scala/Bintray.scala]: We actually want to use the sbt.Logger. Can you add a parameter to `def ensuredCredentials` called `log: Logger`? Then in BintrayPlugin.scala:99 add an argument `streams.value.log` to pass the instance of the logger?

File: src/main/scala/Bintray.scala [hunk 3]
```diff
@@ -60,11 +60,11 @@ object Bintray {
...
           saveBintrayCredentials(credsFile)(requestCredentials())
           ensuredCredentials(credsFile, prompt)
         } else {
-          println(s"Missing bintray credentials $credsFile. Some bintray features depend on this.")
+          log.info(s"Missing bintray credentials $credsFile. Some bintray features depend on this.")
           None
```
Comment (travissarles) [inline src/main/scala/Bintray.scala]: Same here. In this case you could do a log.warn.

Comment (jvican) [inline src/main/scala/Bintray.scala]: 👍

File: src/main/scala/Resolver.scala (hunk 1 above)
Comment (travissarles) [inline src/main/scala/Resolver.scala]: Yes
